clients/unshaded/src/main/java/tachyon/hadoop/ConfUtils.java

```diff
@@ -73,9 +74,21 @@ public static TachyonConf loadFromHadoopConfiguration(Configuration source) {
         LOG.error("Unable to load TachyonConf from Hadoop configuration", e);
         throw new RuntimeException(e);
       }
-
-      return tachyonConfProperties != null ? new TachyonConf(tachyonConfProperties) : null;
     }
-    return null;
+    if (tachyonConfProperties == null) {
+      tachyonConfProperties = new Properties();
+    }
+    // Load any Tachyon configuration parameters existing in the Hadoop configuration.
+    for (Map.Entry<String, String> entry : source) {
+      String propertyName = entry.getKey();
```

Comment:
oh, but for unit test it seems we can test the logic by mocking Configuration `source`